x/leverage/client/cli/query.go

```diff
@@ -95,6 +96,31 @@ func GetCmdQueryRegisteredTokens() *cobra.Command {
 	return cmd
 }
 
+// GetCmdQuerySpecialAssetPairs creates a Cobra command to query for all
+// the special asset pairs in the x/leverage module.
+func GetCmdQuerySpecialAssetPairs() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "special-pairs",
+		Args:  cobra.NoArgs,
```

Comment:
You added an optional `denom` field in the proto query. Shouldn't it be added here as well?

Reply:
good catch - added it